lib/Settings/Admin/AdminSettings.php

```diff
@@ -146,6 +147,11 @@ protected function initMatterbridge(): void {
 			'matterbridge_enable',
 			$this->serverConfig->getAppValue('spreed', 'enable_matterbridge', '0') === '1'
 		);
+
+		$this->initialState->provideInitialState(
+			'public_rooms_allowed',
+			$this->serverConfig->getAppValue('spreed', 'public_rooms_allowed', 'yes') === 'yes'
```

Comment:
We need to expose this to the mobile clients as a capability so they can also hide the features accordingly.

Reply:
This mod was created in web UI in mind. Not experienced in nextcloud devel nor using mobile app so please suggest required changes if necessary.

Reply:
Basically add a line like this which in addition also checks your config:
https://github.com/nextcloud/spreed/blob/91d3bd8df8a5288c2abb3badde00c248d974dae8/lib/Capabilities.php#L151

Reply:
Not experienced with capabilities in Nextcloud - please create change suggestion and will just accept it to this PR or make separate PR to adjust it.